README.md

```diff
@@ -1,2 +1,15 @@
-# node-proxy-server
-This is a project for learning Node/Express. In fact it will be proxy server for original NASA API.
+## About The Project
+This is a project for learning Node/Express. In fact it will be a proxy server for the original NASA API.
```

Comment:
I think the link to NASA API will be useful here

Reply:
Thanks. I've already added these changes to the following PR.